x-pack/plugins/cases/public/components/app/use_available_owners.test.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { renderHook } from '@testing-library/react-hooks';
+
+import { useKibana } from '../../common/lib/kibana';
+import { useAvailableCasesOwners } from './use_available_owners';
+
+jest.mock('../../common/lib/kibana');
+
+const useKibanaMock = useKibana as jest.MockedFunction<typeof useKibana>;
+
+const hasAll = {
+  securitySolutionCases: {
+    crud_cases: true,
+    read_cases: true,
+  },
+  observabilityCases: {
+    crud_cases: true,
+    read_cases: true,
+  },
+};
+
+const hasSecurityAsCrud = {
+  securitySolutionCases: {
+    crud_cases: true,
+  },
+  observabilityCases: {
+    read_cases: true,
+  },
+};
+
+const hasObservabilityAsRead = {
+  observabilityCases: {
+    read_cases: true,
+  },
+  securitySolutionCases: {
+    crud_cases: true,
+  },
+};
+
+const mockKibana = (permissionType: unknown = hasAll) => {
+  useKibanaMock.mockReturnValue({
+    services: {
+      application: {
+        capabilities: permissionType,
+      },
+    },
+  } as unknown as ReturnType<typeof useKibana>);
+};
+
+describe('useKibanaMock', () => {
```

Comment:
fixed! the only thing would be a situation where there is another capability that ends with cases, something like `reportingSolutionCases` As that would get picked up by the hook because it ends with 'cases'. presumably though, if that were to exist, it should be picked up by this hook. If thats not the case we may need to hard code in the solutions it should look for.